scripts/kdb.fish

```diff
@@ -241,8 +241,8 @@ function __fish_kdb_print_resolver_plugins -d 'Print a list of available resolve
 end
 
 function __fish_kdb_print_storage_plugins -d 'Print a list of available storage plugins'
-    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json ni passwd regexstore
-    set -l formats $formats simpleini simplespeclang tcl xmltool uname
+    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json mini ni passwd
+    set -l formats $formats regexstore simpleini simplespeclang tcl xmltool uname
```

Comment:
We would need to profile it to find out where the bottleneck is.

By design `kdb list` needs to load every plugin (even if it does not output all of them). On my computer about 2/3 of `kdb list storage` is spent loading all modules (kdb::tools::Modules::load, and 1/3 in `dlopen`) and afaik module loading is implemented completely different on mac os x. But that is a wild guess, we would need profile data from your machine to find out.

Is it still a usability problem? 200ms (5x 40ms) should still be barely noticeable?